src/navability/services/Variable.jl

```diff
@@ -115,17 +115,37 @@ function SessionKey(
   )
 end
 
-function VariableWhere(
-        sessionKey,
-        label::AbstractString,
-        variableType::VariableType
-    )
-    #
-    Dict{String,Any}(
-        "sessionKey" => sessionKey,
-        "label" => label,
-        "variableType" => variableType
-    )
+function VariableKey(
+    userId::AbstractString,
+    robotId::AbstractString,
+    sessionId::AbstractString,
+    variableLabel::AbstractString
+  )
+  #
+  Dict{String,String}(
+    "userId" => userId,
+    "robotId" => robotId,
+    "sessionId" => sessionId,
+    "variableLabel" => variableLabel
+  )
+end
+
+function VariableId(
+    key
+  )
+  #
+  Dict{String,Any}(
+    "key" => key
+  )
+end
+
+function VariableId(
```

Comment:
whats the difference 2 of 2?

Reply:
Ya that is dup. You removed the correct one.

Reply:
Yay

![image](https://user-images.githubusercontent.com/6412556/190486315-7c3fda80-0be2-4514-97c7-9ccf60608cfc.png)